sdk/core/azure-core/inc/azure/core/version.hpp

```diff
@@ -15,7 +15,7 @@
 #define AZURE_CORE_VERSION_PATCH 0
 #define AZURE_CORE_VERSION_PRERELEASE "beta.4"
 
-namespace Azure { namespace Core {
+namespace Azure { namespace Core { namespace Details {
```

Comment:
Add a changelog entry as this is breaking. We are effectively making these version strings "private".